code/modules/atmospherics/environmental/LINDA_fire.dm

```diff
@@ -50,7 +50,7 @@
 	mouse_opacity = MOUSE_OPACITY_TRANSPARENT
 	icon = 'icons/effects/fire.dmi'
 	icon_state = "1"
-	layer = ABOVE_OPEN_TURF_LAYER
+	layer = FLY_LAYER + 0.1//ABOVE_OPEN_TURF_LAYER
```

Comment:
RIPPLE_LAYER

Reply:
what the fuck is a hyperspace ripple
nevermind give me a second